### PR TITLE
Adding a new phrase to the story.

### PR DESCRIPTION
**Pull request by FranklinLudgood**
Please review this, @casey-collab

### CODE REVIEW

**Comment by FranklinLudgood** [comment]
Please review this, @casey-collab

File: english/Booyah/booyah.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I finally understand git and gitHub. I actually understand branches and merges.
```
Comment (casey-collab) [inline english/Booyah/booyah.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

Comment (casey-collab) [inline english/Booyah/booyah.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by FranklinLudgood** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :rabbit:

**Pull request merged**